app/routes/apply/new-eligibility/about-you.js

```diff
@@ -67,7 +99,7 @@ module.exports = function (router) {
                 nextPage = 'new-claim/past'
               }
 
-              return res.redirect(`/apply/${req.params.claimType}/eligibility/${req.params.referenceId}/${nextPage}`)
+              return res.redirect(`/apply/${claimType}/eligibility/${referenceId}/${nextPage}`)
```

Comment:
happy for this to be the cut off point for moving the path params into session and do the other routes in another PR